hostscripts/clouddata/syncSLErepos

```diff
@@ -99,7 +99,7 @@ rsync_and_unpack_iso()
 for version in 9 8 7 6; do
```

Comment:
You have to add `10` for the conditional below to every work.

Reply:
I intentionally left out 10 because there is no Cloud 10 setup. the point of changing the conditional below to 10 is to indicate that cloud 9 got released and that it shouldn't be syncing the latest build from IBS anymore.

Reply:
Ok, so it's a placeholder?